clog/clog.go

```diff
@@ -0,0 +1,196 @@
+/*
+Package clog provides Conext with logging information.
+*/
+package clog
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/golang/glog"
+)
+
+// unique type to prevent assignment.
+type clogContextKeyT struct{}
+
+var clogContextKey = clogContextKeyT{}
+
+const (
+	// standard keys
+	manifestID    = "manifestID"
+	sessionID     = "sessionID"
+	nonce         = "nonce"
+	seqNo         = "seqNo"
+	orchSessionID = "orchSessionID" // session id generated on orchestrator for broadcaster
+)
+
+// Verbose is a boolean type that implements Infof (like Printf) etc.
+// See the documentation of V for more information.
+type Verbose bool
+
+var stdKeys map[string]bool
+var stdKeysOrder = []string{manifestID, sessionID, nonce, seqNo}
+
+func init() {
+	stdKeys = make(map[string]bool)
+	for _, key := range stdKeysOrder {
+		stdKeys[key] = true
+	}
+}
+
+func V(level glog.Level) Verbose {
+	return Verbose(bool(glog.V(level)))
```

Comment:
```suggestion
	return Verbose(glog.V(level))
```

Nit: You don't need `bool()` type conversion here.

Reply:
thanks, fixed